### PR TITLE
feat: Allow changing languages

### PR DESCRIPTION
**Pull request by Ushie**
- [x] Update the branches in the workflow file
- [x] Translation file post-processor to remove empty strings
- [x] Look into build errors

### CODE REVIEW

**Comment by validcube** [comment]
@Ushie is there anything else's left?

if so please edit this comment or PR description to include another todo list.

**Comment by oSumAtrIX** [comment]
Whats the state of this PR?

**Comment by validcube** [comment]
> Whats the state of this PR?

This PR should be *ready* (aside from fixing conflicts)

But I'm waiting on review because this is a really big changes.

**Comment by validcube** [comment]
@PalmDevs if would like to change some strings, now would be a perfect time to do so.

**Comment by Ushie** [comment]
Updating strings is out of scope

**Comment by validcube** [comment]
String updated: edc8ef4f4449b250018632a4322bdcaac8ebe9e7

File: .github/workflows/sync-crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        with:
+          config: crowdin.yml
+          upload_sources: true
+          upload_translations: false
+          download_translations: true
+          localization_branch_name: feat/translations
+          create_pull_request: true
+          pull_request_title: "chore(i18n): Sync translations"
+          pull_request_body: "Sync translations from [Crowdin/ReVanced](https://crowdin.com/project/revanced)"
+          pull_request_base_branch_name: "feat/i10n"
```
Comment (validcube) [inline .github/workflows/sync-crowdin.yml:31]:
```suggestion
          pull_request_base_branch_name: "dev"
```

File: .github/workflows/sync-crowdin.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          sudo chmod 777 *
+          dart nuke.dart >> $GITHUB_STEP_SUMMARY
+
+      - name: Push out changes to i10n
+        run: |
+          git config user.name revanced-bot
+          git config user.email github@revanced.app
+          sudo chown -R $USER:$USER .git
+          git add assets/i18n/*.json
+          git commit -m "chore(translation): Remove empty values from JSON" assets/i18n/*.json
```
Comment (validcube) [inline .github/workflows/sync-crowdin.yml:50]:
```suggestion
          git commit -m "chore(Translation): Remove empty values from JSON" assets/i18n/*.json
```

**Review by validcube — COMMENTED** [review]
ireallyhopethiswork

File: .github/workflows/sync-crowdin.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+name: Crowdin Action
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [feat/i10n]
```
Comment (validcube) [inline .github/workflows/sync-crowdin.yml:6]: Finale touch


```suggestion
    branches: [dev]
```

**Review by validcube — APPROVED** [review]
Greenlight to review

**Pull request merged**